src/DateTimePicker.jsx

```diff
@@ -184,15 +186,19 @@ class DateTimePicker extends React.Component {
 			// TODO datetime-local opts ?
 
 			return (
-				<DateTimeLocalInput
-					id={id}
-					label={label}
-					value={this.state.datetime}
-					required={required}
-					className={classNames}
-					onChangeCallback={this.setDateTime}
+				<div>
+					<DateTimeLocalInput
+						id={id}
+						label={label}
+						value={this.state.datetime}
+						required={required}
+						className={classNames}
+						onChangeCallback={this.setDateTime}
+						error={error}
```

Comment:
sort of a minor suggestion, but I think it would be nice if `DateTimeLocalInput`, `CalendarComponent`, and `TimeInput` had APIs that were as close to a 'pure HTML' element as possible, meaning that instead of passing `error` (which is _only_ used to set an error `classname`, you should do the classname calculation in the parent and pass the `field--error` class as a `className` prop.

That setup makes the components 'dumber', and more like direct replacements for `<input>`, which I think makes them easier to understand.